kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -397,18 +397,17 @@ impl LookupPattern {
                     .collect()
             }
             LookupPattern::ForeignFieldMul => {
-                (7..=8)
-                    .map(|column| {
-                        //   0 1 2 3 4 5 6 7 8 9 10 11 12 13 14
-                        //   - - - - - - - L L - -  -  -  -  -
-                        JointLookup {
-                            table_id: LookupTableID::Constant(RANGE_CHECK_TABLE_ID),
-                            entry: vec![SingleLookup {
-                                value: vec![(F::one(), curr_row(column))],
-                            }],
-                        }
-                    })
-                    .collect()
+                vec![
+                    //   0 1 2 3 4 5 6 7 8 9 10 11 12 13 14
+                    //   - - - - - - - L - - -  -  -  -  -
```

Comment:
I guess this needs a Mina counterpart to remove the 8 column from it, in particular in this PR: https://github.com/MinaProtocol/mina/pull/12342

Reply:
Correct - it will.